packages/web-app-files/src/views/shares/SharedWithMe.vue

```diff
@@ -20,22 +20,40 @@
             @toggle-filter="setAreHiddenFilesShown"
           />
           <item-filter
-            v-if="shareTypes.length > 1"
```

Comment:
dev leftover that you removed the `v-if`?

Reply:
No, that's a decision we (@tbsbdr and I) made on purpose because it's confusing that the filter just sometimes shows.

Reply:
Ok, thank you for clarifying 👍 makes sense!